src/cuda/unique_reindex.cu

```diff
@@ -214,5 +214,28 @@ std::tuple<torch::Tensor, torch::Tensor> RelabelCUDA(torch::Tensor col_ids,
   torch::Tensor relabeled_indices = ret[1];
   return std::make_tuple(unique_tensor, relabeled_indices);
 }
+
+std::tuple<torch::Tensor, std::vector<torch::Tensor>> RelabelCUDA2(
```

Comment:
What is the difference between `RelabelCUDA` and `RelabelCUDA2`?

Reply:
@gpzlx1

Reply:
Compared `RelabelCUDA`, `RelabelCUDA2` is more flexible and can support relabel on csc or coo format. I will remove the old `RelabelCUDA` and add comments for RelabelCUDA2.